tests/riak_control.erl

```diff
@@ -114,6 +114,7 @@ verify_resource(Node0, Resource) ->
     Node = rt:http_url(Node0),
     Output = os:cmd(io_lib:format("curl -s -S ~s~p", [Node, Resource])),
     lager:info("Verifying node ~p resource ~p.", [Node, Resource]),
+    lager:info("~p~n", [mochijson2:decode(Output)]),
```

Comment:
Not sure we want to leave in this debug output.  If we do, let's label it like in the above line.